pwa/src/components/translations/translationTable.tsx

```diff
@@ -62,10 +52,10 @@ export default function TranslationTable({ tableName }) {
 
   const handleDeleteTranslation = (id): void => {
```

Comment:
This is never used. It should be. 

Maybe you can add a `Delete` button (like in the other forms tables)?